operations/observability/mixins/meta/rules/server.yaml

```diff
@@ -98,17 +98,32 @@ spec:
         dashboard_url: https://grafana.gitpod.io/d/6581e46e4e5c7ba40a07646395ef7b23/kubernetes-compute-resources-pod?var-cluster={{ $labels.cluster }}&var-namespace=default
 
     - alert: WebAppServicesCrashlooping
-      # Reasoning: alert if any pod is restarting more than 3 times / 5 minutes.
-      expr: sum(increase(kube_pod_container_status_restarts_total{container!="POD", pod=~"(content-service|dashboard|db|messagebus|proxy|server|ws-manager-bridge|usage)-.*"}[5m])) by (cluster) > 3
+      # Reasoning: alert if any pod is stuck in crashlooping more than 5 minute.
+      expr: sum(increase(kube_pod_container_status_restarts_total{container!="POD", pod=~"(content-service|dashboard|db|messagebus|proxy|server|ws-manager-bridge|usage)-.*"}[5m])) by (cluster, pod) > 1
+      # Five minutes sound high, but that's the only value that's higher than recent history
       for: 5m
       labels:
-        # sent to the team internal channel until we fine tuned it
+        severity: critical
+        team: webapp
+      annotations:
+        runbook_url: https://github.com/gitpod-io/runbooks/blob/main/runbooks/WebAppServicesCrashlooping.md
+        summary: Pod is crash looping in {{ $labels.cluster }}.
+        description: Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is stuck in crashlooping for at least 5 minutes
+
+    - alert: WebAppServicesCrashloopingInternal
+      # Reasoning: alert if any pod is stuck in crashlooping more than 1 minute.
+      # Used for fine tuning the alert above (WebAppServicesCrashlooping)!
+      expr: sum(increase(kube_pod_container_status_restarts_total{container!="POD", pod=~"(content-service|dashboard|db|messagebus|proxy|server|ws-manager-bridge|usage)-.*"}[5m])) by (cluster, pod) > 1
+      # Let's be more ambitious than 5m
+      for: 1m
+      labels:
+        # sent to the team internal channel until we can propagate these values to CRITICAL
         severity: warning
```

Comment:
Make the internal one even stricter: This applies to _some_ cases over the last months, but those feel too often, so we should fix the root cause. :+1: